### PR TITLE
Show usage on generate email

### PR DESCRIPTION
**Pull request by hcarneiro**

### CODE REVIEW

File: js/libs/builder.js [hunk 1]
```diff
@@ -306,6 +306,14 @@ var app = new Vue({
       var $vm = this;
       var emailProviderData = ($vm.settings && $vm.settings.generateEmailTemplate) || $vm.defaultEmailSettingsForCompose;
 
+      emailProviderData.options = {
```
Comment (squallstar) [inline js/libs/builder.js]: is `emailProviderData` always an object? just wondering if can be undefined.

File: js/libs/builder.js [hunk 2]
```diff
@@ -306,6 +306,14 @@ var app = new Vue({
...
+        usage: {
+          'field-x': 'Insert the value entered in the form field.<br><i>To see the ID of each form field, click to edit the field and the ID can be seen at the top right corner.</i>',
+          appName: 'Insert your app name',
+          organisationName: 'insert your organisation name'
```
Comment (squallstar) [inline js/libs/builder.js]: capital letter on `Insert`?

File: js/libs/builder.js (hunk 1 above)
Comment (hcarneiro) [inline js/libs/builder.js]: Yes it's always a object!

File: js/libs/builder.js (hunk 2 above)
Comment (hcarneiro) [inline js/libs/builder.js]: Fixed

**Pull request merged**